Frontend/ui-common/src/RemoteConfig/Types.purs

```diff
@@ -16,10 +16,10 @@ module Common.RemoteConfig.Types where
 
 import Prelude
 import Data.Maybe (Maybe)
-import Foreign.Class (class Decode, decode)
+import Foreign.Class
 import Data.Generic.Rep (class Generic)
 import Data.Newtype (class Newtype)
-import Presto.Core.Utils.Encoding (defaultDecode)
+import Presto.Core.Utils.Encoding
```

Comment:
try not to remove the imports and use aliases for reference, code readability will be good in that sense